src/content/en/updates/2020/10/http-cache-partitioning.md

```diff
@@ -205,6 +205,31 @@ occurs.
 
 ## FAQs
 
+### Is it already enabled on my Chrome? How can I check?
+
+The feature is being rolled out through late 2020. To check whether your Chrome instance is
+already in effect, follow the instructions below:
+
+1. Open `chrome://net-export/` and press **Start Logging to Disk**.
+2. Specify where to save the log file on your computer.
+3. Browse the web on Chrome for a minute.
+4. Go back to `chrome://net-export/` and press **Stop Logging**.
+5. Go to `https://netlog-viewer.appspot.com/#import`.
+6. Press **Choose File** and pass the log file you saved.
+7. You will see the output of the log file.
```

Comment:
```suggestion
6. Press **Choose File** and pass the log file you saved. 

You will see the output of the log file.
```

Reply:
The best practice is that steps only begin with a user action. Responses to steps are tacked at the end of the step. The result of the whole procedure is in a separate paragraph at the end.